bluetooth/src/commonMain/kotlin/Bluetooth.kt

```diff
@@ -256,6 +258,17 @@ suspend fun Flow<Device?>.updateRssi() {
     }.first()
 }
 
+suspend fun Flow<Device?>.requestMtu(mtu: Int): Boolean {
+    return state().transformLatest { deviceState ->
+        when (deviceState) {
+            is Connected -> {
+                emit(deviceState.requestMtu(mtu))
+            }
+            else -> {}
```

Comment:
In iOS is preferred to have the whole enum in the switch in order to avoid missing case when they are added later. What is the pattern in Kotlin?

Reply:
I think in case it's fine, followed patter in method above (updateRssi) for code consistency

Reply:
👍

Reply:
having an empty else (or an else at all is an anti-pattern. You want the compiler to give an error when you have missing cases. So copied something bad here :)

Reply:
https://github.com/splendo/kaluga/issues/493